org.jellyfin.JellyfinDesktop.yml

```diff
@@ -0,0 +1,171 @@
+app-id: org.jellyfin.JellyfinDesktop
+runtime: org.kde.Platform
+runtime-version: '6.10'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '6.10'
+command: jellyfin-desktop
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --own-name=org.mpris.MediaPlayer2.JellyfinDesktop.*
+cleanup:
+  - '*.a'
+  - '*.la'
+  - /bin/event_rpcgen.py
+  - /include
+  - /lib/cmake
+  - /lib/metatypes
+  - /lib/modules
+  - /lib/pkgconfig
+  - /lib/sbom
+  - /mkspecs
+  - /share/man
+modules:
+  - name: jellyfin-desktop
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DUSE_STATIC_MPVQT=OFF
+    post-install:
+      - install -Dm644 resources/meta/org.jellyfin.JellyfinDesktop.desktop /app/share/applications/org.jellyfin.JellyfinDesktop.desktop
+      - install -Dm644 resources/images/icon.png /app/share/icons/hicolor/256x256/apps/org.jellyfin.JellyfinDesktop.png
+      - install -Dm644 resources/images/icon.svg /app/share/icons/hicolor/scalable/apps/org.jellyfin.JellyfinDesktop.svg
+      - rm -f /app/share/metainfo/org.jellyfin.JellyfinDesktop.appdata.xml
+      - install -Dm644 org.jellyfin.JellyfinDesktop.metainfo.xml /app/share/metainfo/org.jellyfin.JellyfinDesktop.metainfo.xml
```

Comment:
why is this not being done by the upstream build system?

Reply:
This override will only be here for this initial release. I forgot a changes in the metainfo, but didn't want to tag a new release due to other unrelated changes.

---

If it's a blocker, I _can_ release a new version and remove, but I'd strongly prefer to not yet do that.